index.html

```diff
@@ -0,0 +1,26 @@
+<doctype <!DOCTYPE html>
```

Comment:
@ermajinkles Your markup is structured well, however this should be `<!doctype html>` 🔨